src/Cake.Common/Tools/GitVersion/GitVersionAliases.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.GitVersion
+{
+    /// <summary>
+    /// Contains funcionality related to GitVersion.
+    /// </summary>
+    [CakeAliasCategory("GitVersion")]
```

Comment:
@devlead with regard to the above comment that I made, I can't actually see how this would work anymore.  I thought I did, but now I am not so sure.  Is there a way to group all these tools under the GitTools alias, but still also group by both GitReleaseManager and GitVersion?